src/clarity-angular/popover/dropdown/dropdown-item.ts

```diff
@@ -16,7 +16,11 @@ export class DropdownItem {
     @HostListener("click")
     onDropdownItemClick(): void {
         if (this.dropdown.isMenuClosable && !this.el.nativeElement.classList.contains("disabled")) {
-            this.ifOpenService.open = !this.ifOpenService.open;
```

Comment:
actually after discussing with @youdz I am changing the solution completely. Closing this PR